### DISCUSSION TITLE
組み込み系の開発製品の日本語入力にMOZCのコードとキーボードの配置を真似たもの（一部改変）を使用することはライセンス的に可能でしょうか？

### DISCUSSION THREAD

**Discussion by iwanaga0207** [google/mozc#926] [Q&A]
組み込み系の開発製品の日本語入力にMOZCのコード（一部改変）とスクリーンキーボードのキー配置を真似たもの（一部改変）を使用することはライセンス的に可能でしょうか？

どこに問い合わせたらいいのかわからずここで質問させていただきます

よろしくお願いします

**Comment by hiroyuki-komatsu** [comment]
For this kind of question, please talk with your product council in your organization.
Best regards,
